pkg/portutil/portutil.go

```diff
@@ -100,7 +100,17 @@ func ParseFlagP(s string) ([]gocni.PortMapping, error) {
 		if err != nil {
 			return nil, fmt.Errorf("invalid hostPort: %s", hostPort)
 		}
+		usedPorts, err := getUsedPorts(ip, "tcp")
+		if err != nil {
+			return nil, err
+		}
+		for i := startHostPort; i <= endHostPort; i++ {
+			if usedPorts[i] {
+				return nil, fmt.Errorf("ports are not available: exposing port %s %d in use", proto, i)
```

Comment:
Refer to docker errors
```suggestion
				return nil, fmt.Errorf("Bind for %s:%d failed: port is already allocated.", ip, port)
```

Reply:
Updated.